inc/fields/actorfield.class.php

```diff
@@ -70,6 +71,10 @@ public function displayField($canEdit = true) {
                      }
                   })
                   $("#actor_formcreator_field_' . $this->fields['id'] . '").select2("readonly", ' . $readonly . ');
+                  $("#actor_formcreator_field_' . $this->fields['id'] . $rand . '").on("change", function(e) {
```

Comment:
bugfix: the actors were not sent to the server when evaluating visibility of fields